client/landing/jetpack-cloud/sections/scan/main.jsx

```diff
@@ -3,22 +3,103 @@
  */
 import React, { Component } from 'react';
 import { connect } from 'react-redux';
+import { Button } from '@automattic/components';
+import { translate } from 'i18n-calypso';
 
 /**
  * Internal dependencies
  */
-import { getSelectedSiteId } from 'state/ui/selectors';
+import { getSelectedSiteId, getSelectedSiteSlug } from 'state/ui/selectors';
+import { withLocalizedMoment } from 'components/localized-moment';
+import SecurityIcon from 'landing/jetpack-cloud/components/security-icon';
+
+import './style.scss';
 
 class ScanPage extends Component {
+	renderScanOkay() {
+		const { siteSlug, moment, lastScanTimestamp, nextScanTimestamp } = this.props;
+
+		let nextScanString = translate( 'tomorrow' );
```

Comment:
To make it clearer what the case in the `if` statement is, what do you think of this?

```
const isNextScanToday = moment( nextScanTimestamp ).isSame( Date.now(), 'd' );
const nextScanString = isNextScanToday ? translate( 'tomorrow' ) : translate( 'later today' );
```

Reply:
I removed this code in ee35e139de69f85744dbb761641e083cdfcaebdb, but that looks good if we re-add it!